### PR TITLE
Integrate footer into article

### LINKED ISSUE #139: Integrate Footer to Article component
**Is your feature request related to a problem? Please describe.**
This should be only carried out after all the other footer tickets have been completed.

The Footer is not integrated into the Article component, and by extension not shown on the page.

**Describe the solution you'd like**
Integrate the [Footer component](https://github.com/bbc/simorgh/blob/latest/src/app/components/Footer/index.jsx) with the [Article component]( https://github.com/bbc/simorgh/blob/latest/src/app/components/Article.jsx#L38-L44).

**Additional context**
- Will need to have all accessibility requirements addressed, working closely with a UX designer.
- Will likely need to add e2e tests, working closely with a QA tester.

**Definition of done:**
- [ ] Footer to have <footer> element wrapping the Brand banner and footer links
Footer Links section:
- [ ] Copy and links: Brevier - #444444
- [ ] Copy and links BBCReithSans-Regular (to be discussed) 
- [ ] Background colour: #4C4C4C
- [ ] Links: weight - 700 
- [ ] Dividers: 1px height #ffffff
- [ ] Tap targets (Trust Project Link: 32px height, 100% width. Other l inks: 32px height, 50% width (mobile only), greater than mobile:  24px
- [ ] Tab Order: Including banner (banner to be no.1)
- [ ] Landmark region content info
- [ ] Announced Text of links - (To discuss with Catherine about BBC News SVG)

### PR DESCRIPTION
**Pull request by jtart**
Resolves #139

_Integrates Footer container into Article component.._

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by sareh — DISMISSED** [review]
👍 Looks good. I'd suggest that it's worth ignoring the CodeClimate duplication warning here.

**Review by PaulChinkwende — DISMISSED** [review]

File: cypress/integration/footerSpec.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  shouldContainStyles,
+} from '../support/testHelper';
+
+describe('Footer Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    // Only 'c0000000025o' & 'c0000000027o' are available within the PROD enviroment
+    cy.visit('/news/articles/c0000000025o');
+  });
+  it('should render the BBC News branding', () => {
```
Comment (jtart) [inline cypress/integration/footerSpec.js:15]: Aren't these tests essentially just what is tested on the header? We should re-use all of the code.

File: cypress/support/testHelper.js [hunk 2]
```diff
@@ -25,6 +25,30 @@ export const checkElementStyles = (elementString, text, color, fontFamily) => {
   shouldContainStyles(el, 'font-family', fontFamily);
 };
 
+export const checkFooterLinks = (position, url) => {
```
Comment (jtart) [inline cypress/support/testHelper.js:28]: These test helpers are quite ambiguous, and could be namespaced to footer helpers.

**Comment by jtart** [comment]
I can't officially review as it is my PR. I have some changes, but in the interest of getting stuff through, I am happy to merge.

File: cypress/integration/footerSpec.js (hunk 1 above)
Comment (sareh) [reply cypress/integration/footerSpec.js:15]: Yes, they're the same, other than the element name. Agreed on re-using the existing code.

**Pull request merged**